README.md

```diff
@@ -53,18 +53,15 @@ chmod +x minio
 ```
 
 ### Snap
-You can install the latest `minio` [snap](https://snapcraft.io), and help testing the most recent changes of the master branch in [all the supported Linux distros](https://snapcraft.io/docs/core/install) with:
-
+You can install the latest `minio` [snap](https://snapcraft.io).
```

Comment:
It is better to make it consistent with other install managers like HomeBrew above. So we can just say

`Install minio using [Snap](https://snapcraft.io)`